lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -70,12 +70,29 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 		return await this.driver.findElement( textSelector ).sendKeys( text );
 	}
 
-	async insertContactForm() {
+	async insertShortcode( shortcode ) {
```

Comment:
I like this solution 👍I think there is no need to make separate PR only for this, but it would be useful in the future definitely.